extensions/wikia/GlobalFooter/GlobalFooterController.class.php

```diff
@@ -80,6 +86,8 @@ private function getGlobalFooterLinks() {
 			}
 		}
 
+		$wgMemc->set( $memcKey, $parsedLinks, self::MEMC_EXPIRY );
```

Comment:
There's a slight possibility that we cache an empty response i.e. when:
* there is no message in Memcache
* AND `getMessageAsArray()` finds a message, but its content is empty (`is_null($var)` is equal to `$var === NULL`, so an empty response is still not a `NULL`—lines 65-66).
But that may be also intended. :confused: